### PR TITLE
Fix/frontend78 search page

### PR DESCRIPTION
**Pull request by Barracuda713**
- Fixed known bugs with search filters
- add header mobile search input
- add video card (for main) component (based on search video card)

upd: 
- set updateTimer for VideoCard to 1 minute in search and 5 minutes in main
- date dependent timer delay will be added  in next PRs

### CODE REVIEW

File: frontend/src/components/search/components/video-card/styles.module.scss [hunk 1]
```diff
@@ -63,28 +67,80 @@
...
+  z-index: 15;
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  gap: 10px;
+  padding: 6px 15px;
+  color: var(--white-text-color);
+  font-size: 10px;
+  line-height: 14px;
+  background-color: rgb(0 0 0 / 70%);
```
Comment (foxt451) [inline frontend/src/components/search/components/video-card/styles.module.scss:83]: add the color to variables

File: frontend/src/components/search/components/video-card/styles.module.scss [hunk 2]
```diff
@@ -63,28 +67,80 @@
...
+.video-card-badge-scheduled-btn {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  gap: 10px;
+  height: 24px;
+  margin-left: 15px;
+  padding: 4px 10px;
+  color: var(--white-text-color);
+  background-color: rgb(255 255 255 / 25%);
```
Comment (foxt451) [inline frontend/src/components/search/components/video-card/styles.module.scss:109]: add the color to variables

File: frontend/src/components/search/components/video-card/styles.module.scss [hunk 3]
```diff
@@ -111,14 +167,36 @@
...
 
+.video-card-meta-data-scheduled {
+  display: flex;
+  flex-flow: row nowrap;
+  align-items: center;
+  gap: 3px;
+  height: 20px;
+  padding: 3px 6px 3px 3px;
+  color: var(--white-text-color);
+  background-color: rgb(255 255 255 / 15%);
```
Comment (foxt451) [inline frontend/src/components/search/components/video-card/styles.module.scss:188]: add the color to variables

File: frontend/src/components/search/components/video-card/styles.module.scss [hunk 4]
```diff
@@ -111,14 +167,36 @@
...
+  border-radius: 3px;
+  cursor: pointer;
+}
+
+.video-card-meta-data-scheduled svg path {
+  fill: var(--white-text-color);
+}
+
+.video-card-meta-data-scheduled:hover {
+  background-color: rgb(255 255 255 / 25%);
```
Comment (foxt451) [inline frontend/src/components/search/components/video-card/styles.module.scss:198]: add the color to variables

File: frontend/src/components/search/search.tsx [hunk 5]
```diff
@@ -43,11 +43,12 @@ const Search: FC = () => {
       }
       return prev;
     }, {});
-  };
+    // eslint-disable-next-line
```
Comment (foxt451) [inline frontend/src/components/search/search.tsx:46]: it doesn't seem right

**Review by foxt451 — CHANGES_REQUESTED** [review]

**Comment by Barracuda713** [comment]
Due to a lot of merge conflicts and almost completely updated header component, 
I decided to close this PR and make a new one with uptodate components.

**Pull request closed without merging**